pkg/master/master.go

```diff
@@ -157,6 +146,13 @@ type Master struct {
 	serviceReadWritePort int
 	masterServices       *util.Runner
 
+	storage map[string]apiserver.RESTStorage
+
+	nodeRegistry      minion.Registry
```

Comment:
The more they flatten the less interplay there should be (other than reuse by loop code for efficiency)


> On Feb 13, 2015, at 6:23 PM, Brian Grant <notifications@github.com> wrote:
> 
> In pkg/master/master.go:
> 
> > @@ -157,6 +146,13 @@ type Master struct {
> >  	serviceReadWritePort int
> >  	masterServices       *util.Runner
> >  
> > +	storage map[string]apiserver.RESTStorage
> > +
> > +	nodeRegistry      minion.Registry
> Hmm. Ok. I see their other uses. I suppose this is better than a map of registries in that it let's us know which are actually used.
> 
> —
> Reply to this email directly or view it on GitHub.
>